test/scripts/expected_skv2_result.test

```diff
@@ -264,3 +264,9 @@ message VirtualServiceStatus {
 	// Opaque details about status results
```

Comment:
do we want to be committing this .test file?

Reply:
this existed before. It was a test mostly used to confirm that skv2 codegen doesn't change underneath us. My changes did change it, so the test result had to be updated as well